lumicks/pylake/population/detail/fit_info.py

```diff
@@ -10,8 +10,42 @@ class FitInfo:
 
 
 @dataclass(frozen=True)
-class HmmFitInfo(FitInfo):
+class GmmFitInfo(FitInfo):
+    r"""
+    Fitting information for a HiddenMarkovModel
+
+    Parameters
+    ----------
+    converged: bool
+        Whether the model converged
+    n_iter: int
+        Number of iterations during the fit
+    bic: float
+        Bayesian Information Criterion (BIC) on fitted data
+
+        .. math::
+            BIC = k \ln{(n)} - 2 \ln{(L)}
+
+        Where :math:`k` is the number of fitted parameters, :math:`n` is the number of observations
+        (data points) and :math:`L` is the maximized value of the likelihood function
+    aic: float
+        Akaike Information Criterion on fitted data
+
+        .. math::
+            AIC = 2 k - 2 \ln{(L)}
+
+        Where :math:`k` is the number of fitted parameters, :math:`n` is the number of observations
+        (data points) and :math:`L` is the maximized value of the likelihood function
+    lower_bound: float
+        Lower bound value on the log likelihood
```

Comment:
Would be nice to add a bit more description on how to interpret this `lower_bound`. Right now, just saying `lower bound value on the log likelihood` doesn't tell users much. How are they supposed to interpret this value?

Reply:
this is the extent of the description in the `sklearn` docs for the model that we wrap, so I don't really want to go into any more detail. Could add a link to their docs for more info?

Reply:
from the `sklearn` docs I think it's because they have an option for a variational model where instead of optimizing on the log likelihood you optimize on the lower bound for the evidence; so this name seems to be a catch-all. I wonder if we should just use `log_likelihood` instead and then we don't need 2 separate dataclasses to differentiate between gmm and hmm